board/safety/safety_gm.h

```diff
@@ -187,6 +193,22 @@ static int gm_tx_hook(CAN_FIFOMailBox_TypeDef *to_send) {
     if (violation) {
       tx = 0;
     }
+
+    //Check that approved LKAS message has correct rolling counter value and isn't arriving too soon
+    if (tx == 1) {
```

Comment:
Using tx rather than violation just in case tx would ever be 0 without a violation. Since there are other, earlier operations setting the tx flag that may use different logic from the violation flag, this seems necessary.